detector.js

```diff
@@ -21,24 +23,38 @@ function getHandPoints(predictions) {
     return pointsData.flat();
 }
 
-function getShortestDistance(points1, points2) {
-    // TODO better algorithm
-    let shortestDistance = undefined;
-    for (let i = 0; i < points1.length; i++) {
-        for (let j = 0; j < points2.length; j++) {
-            const p1 = points1[i];
-            const p2 = points2[j];
-            const x = p1[0] - p2[0];
-            const y = p1[1] - p2[1];
-            const z = p1[2] - p2[2];
-            const d = Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2) + Math.pow(z, 2));
-            if (shortestDistance === undefined || d < shortestDistance.d) {
-                shortestDistance = { x, y, z, d };
+function getShortestDistance(handPoints, facePoints, distance_threshold) {
```

Comment:
For `distance_threshold` and other variables in the file, use camel case instead as the convention.

Reply:
Maybe we can also have a benchmark of these two and see if the new one is actually faster? Since I assume there is a bunch more memory allocation with the octree

Reply:
reverted back to the original algorithm